themes/mainroad/layouts/index.html

```diff
@@ -6,6 +6,14 @@
 			England.
 		</p>
 
+		<h2>RSE Midlands 2025</h2>
+		<p>
+			RSE Midlands 2025 will be hosted by the the University of Birmingham <a href="">Advanced Research Computing team </a> on monday the 7th of april 2025. At <a href="https://conferences.bham.ac.uk/venues/the-exchange/">The Exchange, 3 Centenary Square, B1 2DR</a>. 		
+               </p>
+		<p>
+			Further information on how to secure tickets will follow in due course..
+		</p>
+
```

Comment:
```suggestion
		<h2>RSE Midlands 2025</h2>
		<p> 
			RSE Midlands 2025 will take place on Monday 7th April 2025 at <a href="https://conferences.bham.ac.uk/venues/the-exchange">The Exchange, 3 Centenary Square, B1 2DR</a>, hosted by the University of Birmingham's <a href="https://www.birmingham.ac.uk/research/arc/team">Advanced Research Computing team</a>.
		</p>
		<p>
			We are seeking <b>expressions of interest</b> from anyone who would like to deliver a lightning talk during the event. Please see our "<a href="https://ukrse.slack.com/archives/C02333HNA64">#midlands</a>" channel in the
			<a href="https://join.slack.com/t/ukrse/signup">RSE Society's slack workspace</a> for full details.
		</p>
		<p>
			Further information on how to secure tickets will follow in due course.
		</p>

```

- tweaking first paragraph for consistency with our slack announcement
- adding hyperlink for ARC
- moving section on lightning talks into here to keep everything in one place

Might be nice if we had a means of contacting us besides "see our `#midlands` channel",  but we don't have a Contact page and I'm not inclined to add my email address to a public site!